### PR TITLE
Update On_Call.md

### PR DESCRIPTION
**Pull request by tashimi**
- this needs more work for current service state information (cc @reset & @fnichol)
- clarifies incident response communication procedures
- clarifies pre-game requirements for on-call 
- removes outdated information about trouble shooting

Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: ON_CALL.md [hunk 1]
```diff
@@ -1,48 +1,61 @@
...
+* You are expected to diagnose, to the best of your ability, and bring in help if needed to resolve.
+
+## Communication Responsibilities
+* Declare an operational issue on statuspage.io. This will automatically post an alert to habitat's slack #general channel, update our statuspage, and tweet.
+* You are responsible for informing your team that you are taking charge of the issue, and you are responsible for telling your team that you are unable to continue responding to that issue before you stop working on it.
+* You are not responsible for responding to questions or community questions. 
+* You are responsible for updating statuspage.io as status changes. 
+* You are not responsible for posting minutes during the issue if status has not materially changed. All updates go into statuspage.io, no other venue.
+* Tone in statuspage.io communication should be friendly and professional.
+* You are responsible for escalating immediately to someone more senior or knowledgeable if you cannot fix the issue alone or need help. Never be afraid to start phoning people, regardless of time or day. Getting production into a good state should be your only concern.
```
Comment (raskchanky) [inline ON_CALL.md]: Do we have people's contact info somewhere?  I can't phone anyone on our team right now if I needed to.

**Comment by mwrock** [comment]
I think that's all in pagerduty.

File: ON_CALL.md (hunk 1 above)
Comment (tashimi) [reply ON_CALL.md]: @raskchanky let's start a spreadsheet

**Pull request merged**